chatbot.py

```diff
@@ -83,8 +79,7 @@ def message_to_bot(H,clf,learn_response):
             if len(proper_nouns) == 2:
                 location_Dict["origin"]=proper_nouns.pop()
                 location_Dict["destination"]=proper_nouns.pop()
```

Comment:
Change to new variable name `location_dict`

Reply:
Have changed it now